libraries/AP_TemperatureSensor/TSYS01.cpp

```diff
@@ -1,8 +1,7 @@
 #include "TSYS01.h"
 
-#include <utility>
 #include <stdio.h>
-#include <AP_HAL/AP_HAL.h>
```

Comment:
Uh.  We're creating an `AP_HAL::HAL` several lines down.  We really ought to include its header.